src/sys/stat.rs

```diff
@@ -66,13 +65,135 @@ libc_bitflags! {
 }
 
 #[cfg(any(target_os = "macos", target_os = "ios", target_os = "openbsd"))]
-pub type type_of_file_flag = c_uint;
+pub type type_of_st_flag = c_uint;
```

Comment:
I renamed `type_of_file_flag` to `type_of_st_flag` to be consistent with other abstraction layer types.

Reply:
Seems like I should rename it to `type_of_st_flags` to match the field name `st_flags` though...